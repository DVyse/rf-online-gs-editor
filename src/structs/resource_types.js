export const PLAYER_BONE1 = 'player_bone1';
export const PLAYER_MESH1 = 'player_mesh1';
export const PLAYER_ANI1 = 'player_ani1';
export const PLAYER_BONE2 = 'player_bone2';
export const PLAYER_MESH2 = 'player_mesh2';
export const PLAYER_ANI2 = 'player_ani2';
export const PLAYER_BONE3 = 'player_bone3';
export const PLAYER_MESH3 = 'player_mesh3';
export const PLAYER_ANI3 = 'player_ani3';
export const MONSTER_BONE = 'monster_bone';
export const MONSTER_MESH = 'monster_mesh';
export const MONSTER_ANI = 'monster_ani';
export const ANIMUS_BONE = 'animus_bone';
export const ANIMUS_MESH = 'animus_mesh';
export const ANIMUS_ANI = 'animus_ani';
export const TOWER_BONE = 'tower_bone';
export const TOWER_MESH = 'tower_mesh';
export const TOWER_ANI = 'tower_ani';
export const STORE_BONE = 'store_bone';
export const STORE_MESH = 'store_mesh';
export const STORE_ANI = 'store_ani';
export const ITEM_BONE = 'item_bone';
export const ITEM_MESH = 'item_mesh';
export const ITEM_ANI = 'item_ani';
export const MAU_BONE = 'mau_bone';
export const MAU_MESH = 'mau_mesh';
export const MAU_ANI = 'mau_ani';
