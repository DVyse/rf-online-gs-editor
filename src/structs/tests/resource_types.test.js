import * as resourceTypes from '../resource_types';

describe('Structs', () => {
  describe('ResourceTypes', () => {
    it('should have resourceTypes', () => {
      expect(resourceTypes).toMatchSnapshot();
    });
  });
});
