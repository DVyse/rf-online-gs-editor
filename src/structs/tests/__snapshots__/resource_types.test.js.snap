// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Structs ResourceTypes should have resourceTypes 1`] = `
Object {
  "ANIMUS_ANI": "animus_ani",
  "ANIMUS_BONE": "animus_bone",
  "ANIMUS_MESH": "animus_mesh",
  "ITEM_ANI": "item_ani",
  "ITEM_BONE": "item_bone",
  "ITEM_MESH": "item_mesh",
  "MAU_ANI": "mau_ani",
  "MAU_BONE": "mau_bone",
  "MAU_MESH": "mau_mesh",
  "MONSTER_ANI": "monster_ani",
  "MONSTER_BONE": "monster_bone",
  "MONSTER_MESH": "monster_mesh",
  "PLAYER_ANI1": "player_ani1",
  "PLAYER_ANI2": "player_ani2",
  "PLAYER_ANI3": "player_ani3",
  "PLAYER_BONE1": "player_bone1",
  "PLAYER_BONE2": "player_bone2",
  "PLAYER_BONE3": "player_bone3",
  "PLAYER_MESH1": "player_mesh1",
  "PLAYER_MESH2": "player_mesh2",
  "PLAYER_MESH3": "player_mesh3",
  "STORE_ANI": "store_ani",
  "STORE_BONE": "store_bone",
  "STORE_MESH": "store_mesh",
  "TOWER_ANI": "tower_ani",
  "TOWER_BONE": "tower_bone",
  "TOWER_MESH": "tower_mesh",
}
`;
